hack/bump_metallb.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+HASH=`git ls-remote https://github.com/metallb/metallb | grep refs/heads/main | cut -f 1`
+sed -i "s/export METALLB_COMMIT_ID=.*/export METALLB_COMMIT_ID=\"$HASH\"/g" hack/common.sh
```

Comment:
Can we also update the value in the .github ci?

Reply:
it's already done by the generate-metallb-manifests.sh

Reply:
oh, I didn't remember about it.